packages/storefront-sdk/src/index.ts

```diff
@@ -1,7 +1,18 @@
-function placeholder(): object {
-	console.log('placeholder');
+import { StorefrontClient } from './client/index.js';
+import { errorMessages } from './utils/index.js';
 
-	return {};
+export interface StorefrontClientParams {
+	/** Nacelle Storefront GraphQL Endpoint. This can be retrieved from the Nacelle Dashboard. */
+	storefrontEndpoint: string;
+
+	/** Optional fetch implementation. If not supplied, the Storefront SDK will use `globalThis.fetch`. */
+	fetchClient?: typeof globalThis.fetch;
 }
 
-export default placeholder;
+export function Storefront(params: StorefrontClientParams) {
+	if (!params?.storefrontEndpoint) {
+		throw new Error(errorMessages.missingEndpoint);
+	}
+
+	return new StorefrontClient(params);
+}
```

Comment:
Do we want to also export this function as the default to match [the v1 sdk](https://github.com/getnacelle/nacelle-storefront-sdk/blob/80626c98938ec3107b6dc9ec31d06ae91b76489a/src/index.ts#L7)?

Reply:
That's certainly a possibility, although I wouldn't mind making the removal of the default export a breaking change. When we use the default export, we get this warning from Vite, as we've seen with other Vite projects with mixed exports:

![Entry module  src:index ts  is using named and default exports together  Consumers of your bundle will have to use `NacelleStorefrontSdk default` to access the default export, which may not be what you want](https://user-images.githubusercontent.com/5732000/210089294-70e35c09-830f-4656-872e-842192917c62.png)

Reply:
Per conversation on Zoom, we'll leave this as a named export for now, but will explore impact of this breaking change and leave the default export as a possibility going forward.